package.json

```diff
@@ -46,7 +46,7 @@
     "appium-chromedriver": "^2.1.3",
     "appium-instruments": "^2.0.5",
     "appium-support": "=1.1.2",
-    "appium-uiauto": "=1.10.8",
+    "appium-uiauto": "=1.10.9",
```

Comment:
Is there a reason that this is locked to `1.10.9` and not just the highest in this version?